std/assembly/internal/array.ts

```diff
@@ -14,7 +14,7 @@ export function defaultComparator<T>(): (a: T, b: T) => i32 {
     return (a: T, b: T): i32 => {
       // JavaScript by default casting all values to string
       // so for emulate this we use trick below
-      // TODO can we improve this?
+      // TODO can we do better?
```

Comment:
I don't think that we should emulate JS behavior there.

Reply:
Yep, by default. But WDYT about add new compiler flag like `--js-compat` which allow backward compatibility?